src/py/flwr/cli/new/templates/app/code/flwr_tune/app.py.tpl

```diff
@@ -1,89 +1,34 @@
 """$project_name: A Flower / FlowerTune app."""
 
 import os
-import warnings
 from datetime import datetime
 
-from flwr_datasets import FederatedDataset
-from hydra import compose, initialize
-from hydra.utils import instantiate
-
 from flwr.client import ClientApp
-from flwr.common import Context, ndarrays_to_parameters
-from flwr.server import ServerApp, ServerAppComponents, ServerConfig
+from flwr.server import ServerApp
 
-from $import_name.client_app import gen_client_fn, get_parameters
-from $import_name.dataset import get_tokenizer_and_data_collator_and_propt_formatting
-from $import_name.models import get_model
-from $import_name.server_app import fit_weighted_average, get_evaluate_fn, get_on_fit_config
+from $import_name.client import gen_client_fn
+from $import_name.server import gen_server_fn
 
 # Avoid warnings
-warnings.filterwarnings("ignore", category=UserWarning)
 os.environ["TOKENIZERS_PARALLELISM"] = "true"
 os.environ["RAY_DISABLE_DOCKER_CPU_WARNING"] = "1"
 
-# Initialise regular config
-with initialize(config_path="conf", version_base="1.1"):
-    cfg = compose(config_name="config")
-
-# Initialise static config
-with initialize(config_path="conf", version_base="1.1"):
-    cfg_static = compose(config_name="static_config")
-
-cfg.train.num_rounds = cfg_static.num_rounds
-
 # Create output directory given current timestamp
 current_time = datetime.now()
 folder_name = current_time.strftime("%Y-%m-%d_%H-%M-%S")
 save_path = os.path.join(os.getcwd(), f"results/{folder_name}")
 os.makedirs(save_path, exist_ok=True)
```

Comment:
Since passing `save_path` is the only reason why we have the `gen_client_fn()` and `gen_server_fn()` I wonder if we could create move these lines inside `server_fn()` and then communicate the `save_path` via the already existing `get_on_fit_config()`. In this way we can (1) get rid off this `app.py` and keep simpler code for `client_fn` and `server_fn`. What do you think?

Reply:
Yes sounds good to me